pkg/proxy/ipvs/proxier.go

```diff
@@ -480,7 +481,7 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 		return nil, err
 	}
 	builtinModsFilePath := fmt.Sprintf("/lib/modules/%s/modules.builtin", kernelVersion)
-	b, err := ioutil.ReadFile(builtinModsFilePath)
+	b, err := ioutil.ReadFile(filepath.Clean(builtinModsFilePath))
```

Comment:
shouldn't this run on kernelVersion before we include it in a filepath, and disallow a cleaned kernelVersion that contains multiple path segments?

Reply:
hi, thanks for your replay. As  a variable, "kernelVersion" maybe begin with "/" or something else.  so i think if i run on kernelVersion firstly,maybe the "builtinModsFilePath" is still not cleaned.
in fact, i did it at the beginning，but still get warning from GoSec.... :(